src/goose/defaults.clj

```diff
@@ -30,6 +34,14 @@
       (str/split (re-pattern (str queue-prefix "*")))
       (second)))
 
+(defn ^:no-doc prefix-batch
+  [batch-id]
+  (str batch-prefix batch-id))
+
+(defn ^:no-doc suffix-batch-set
```

Comment:
Instead of suffix-batch-set, what about batch-set.
`suffix-batch-set` suggests that we are adding suffix to the key of batch-set(more of an implementation detail). Instead a name of what its returning might be more accurate. wdyt?

Reply:
I think it is valuable to be explicit about how the name would look like, which we get with `suffix-` in the fn name. Also, this is in agreement with the other functions of the namespace that prepare name strings.

Reply:
Makes sense. But, currently in Goose, we use `<>-prefix` to construct the prefixes and  `prefixed-<>` as the final  redis key. So we can rename the functions as this
```
(defn ^:no-doc prefixed-batch-set
  [batch-id set]
  (str (prefix-batch batch-id) "/" set))
```
wdyt?

Reply:
`prefixed-<>`: These are `def`s. But we've defined a `defn`.
Ideally fn names should be a verb. Let me know what you think.

Reply:
Okay. @olttwa what do you think of the function name?

Reply:
@siripr4 @alishamohanty how about `suffix-job-set` coz thats we are actually adding as a suffix.

Reply:
Renamed this to `construct-batch-job-set` as discussed on the Slack [thread](https://nilenso.slack.com/archives/C03L1JGP3ME/p1688983812464679)